scripts/spi_flash/board_defs.py

```diff
@@ -128,6 +128,15 @@
         'mcu': "stm32g0b1xx",
         'spi_bus': "spi1",
         "cs_pin": "PB8"
+    },
+    'kingroon-kp3-v1.3-board': {
```

Comment:
I think this would be better called 'mks-robin-nano' since that's what it sounds like the mainboard actually is.